editor/block-settings-menu/index.js

```diff
@@ -1,70 +1,66 @@
 /**
  * External dependencies
  */
+import classnames from 'classnames';
 import { connect } from 'react-redux';
 
 /**
  * WordPress dependencies
  */
 import { __ } from '@wordpress/i18n';
 import { IconButton } from '@wordpress/components';
+import { Component } from '@wordpress/element';
 
 /**
  * Internal dependencies
  */
 import './style.scss';
-import { isEditorSidebarOpened } from '../selectors';
+import BlockSettingsMenuContent from './content';
 import { selectBlock } from '../actions';
 
-function BlockSettingsMenu( { onDelete, onSelect, isSidebarOpened, toggleSidebar, setActivePanel } ) {
-	const toggleInspector = () => {
-		onSelect();
-		setActivePanel();
-		if ( ! isSidebarOpened ) {
-			toggleSidebar();
-		}
-	};
+class BlockSettingsMenu extends Component {
+	constructor() {
+		super( ...arguments );
+		this.toggleMenu = this.toggleMenu.bind( this );
+		this.state = {
+			opened: false,
+		};
+	}
 
-	return (
-		<div className="editor-block-settings-menu">
-			<IconButton
-				className="editor-block-settings-menu__control"
-				onClick={ toggleInspector }
-				icon="admin-generic"
-				label={ __( 'Show inspector' ) }
-			/>
-			<IconButton
-				className="editor-block-settings-menu__control"
-				onClick={ onDelete }
-				icon="trash"
-				label={ __( 'Delete the block' ) }
-			/>
-		</div>
-	);
+	toggleMenu() {
+		this.props.onSelect();
+		this.setState( ( state ) => ( {
+			opened: ! state.opened,
+		} ) );
+	}
+
+	render() {
+		const { opened } = this.state;
+		const { uid } = this.props;
+		const toggleClassname = classnames( 'editor-block-settings-menu__toggle', 'editor-block-settings-menu__control', {
+			'is-opened': opened,
+		} );
+
+		return (
+			<div className="editor-block-settings-menu">
+				<IconButton
+					className={ toggleClassname }
+					onClick={ this.toggleMenu }
+					icon="ellipsis"
+					aria-label={ __( 'Open Settings Menu' ) }
```

Comment:
We should assign this prop as `label`, which has the same effect of applying `aria-label`, but also enables the tooltip to be shown when hovering the icon.

Reply:
Actually, I did it intentionally to avoid the tooltip, I thought it was not necessary here, but I can change.

Reply:
Hmm, I guess it's to a broader question of when and where to apply tooltips. My philosophy has been if there is no adjacent text explaining what the icon is for, that a tooltip is a good idea.